src/locales/en-US/stream.ftl

```diff
@@ -566,7 +566,7 @@ comments-reportPopover-reportThisComment = Report This Comment
 comments-reportPopover-whyAreYouReporting = Why are you reporting this comment?
 
 comments-reportPopover-reasonOffensive = This comment is offensive
-comments-reportPopover-reasonAbusive = This is abusive behaviour
+comments-reportPopover-reasonAbusive = This commenter is being abusive
```

Comment:
I think @wyattjoh indicated that we have to create a new key and not to just update the value of the old key

Reply:
Depends! If we can confirm that the previous english translation was used as the base for other languages, then we need to change the key. If not, then we can keep it the same.

Reply:
We don't need to change the key, because the meaning of the key has not changed at its core. The sole purpose of this translation change was to solve an specific English problem.

Reply:
Works for me